src/main/java/pl/tomaszdziurko/jvm_bloggers/view/login/attack/stream/BruteForceLoginAttackMailSubscriber.java

```diff
@@ -0,0 +1,45 @@
+package pl.tomaszdziurko.jvm_bloggers.view.login.attack.stream;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.tuple.Pair;
+import pl.tomaszdziurko.jvm_bloggers.mailing.MailSender;
+import rx.Subscriber;
+
+/**
+ * @author Adam Dec
+ * @since 0.7.0
+ */
+@Slf4j
+public class BruteForceLoginAttackMailSubscriber extends Subscriber<Pair<String, String>> {
+
+    private static final String ADMIN_EMAIL_ADDRESS = "admin@admin.pl";
```

Comment:
You can use TEST_EMAIL from Settings table. Renaming it to ADMIN_EMAIL or something similar seems a good idea too :)